extension/httpfs/http_state.cpp

```diff
@@ -82,6 +83,29 @@ shared_ptr<HTTPState> HTTPState::TryGetState(optional_ptr<FileOpener> opener, bo
 	return nullptr;
 }
 
+void HTTPState::WriteProfilingInformation(std::ostream &ss) {
+	string read = "in: " + StringUtil::BytesToHumanReadableString(total_bytes_received);
+	string written = "out: " + StringUtil::BytesToHumanReadableString(total_bytes_sent);
+	string head = "#HEAD: " + to_string(head_count);
+	string get = "#GET: " + to_string(get_count);
+	string put = "#PUT: " + to_string(put_count);
+	string post = "#POST: " + to_string(post_count);
+
+	constexpr idx_t TOTAL_BOX_WIDTH = 39;
+	ss << "┌─────────────────────────────────────┐\n";
+	ss << "│┌───────────────────────────────────┐│\n";
```

Comment:
Since this is now no longer shared between azure and httpfs, think the user would need to be informed somehow that these stats are related to a specific extension. Perhaps this box needs to be annotated with the extension name? Then we can do the same for azure and will simply have 2 http stats boxes for both extensions

Reply:
Yea that works, that's what I meant with the first proposed solution, but good point about clarifying which extension it belongs to 👍